app/assets/javascripts/oxalis/model/helpers/proto_helpers.js

```diff
@@ -24,7 +22,12 @@ export function parseProtoTracing(
   const protoRoot = Root.fromJSON(PROTO_FILES[annotationType]);
   const messageType = protoRoot.lookupType(PROTO_TYPES[annotationType]);
   const message = messageType.decode(new Uint8Array(tracingArrayBuffer));
-  return messageType.toObject(message, { arrays: true, objects: true, enums: String });
+  return messageType.toObject(message, {
+    arrays: true,
+    objects: true,
+    enums: String,
+    longs: Number,
```

Comment:
When decoding the protobuf object in node (in the tests), some kind of LONG numbers were created for values that are typed as `int64` in the protobuf definition. I've never seen that before and it looked kind of weird ^^
Like this:
```
{
  version: LONG {
    low: 1
    high: 123
    negative: true
  }
}
```
Have you seen this before? Anyway, I declared that longs should be interpreted as numbers here, therefore. To my knowledge chrome doesn't support longs yet anyways, so I'm not sure what chrome did there, before.